### PR TITLE
Feat/rapid 3461 improve examples

### PR DESCRIPTION
**Pull request by LinoGiger**
-adjusted the examples to be more engaging
-renamed transcription to select words
-added ability to upload audio and video from links

### CODE REVIEW

**Review by corkillj — APPROVED** [review]
LGTM

File: docs/quickstart.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# Quickstart Classification Guide
+# Quickstart Guide
 
 Directly ask real humans to classify your data. This guide will show you how to create a classification order using the Rapidata API.
 
+There are many other types of orders you can create which you can find in the examples.
```
Comment (Makuh17) [inline docs/quickstart.md:5]: Should we link to the examples?

File: src/rapidata/rapidata_client/assets/media_asset.py [hunk 2]
```diff
@@ -21,68 +22,76 @@ class MediaAsset(BaseAsset):
...
+            raise ValueError("Media must be a string, either a local file path or a URL")
 
         if re.match(r'^https?://', path):
-            self.path = MediaAsset.get_image_bytes(path)
+            self.path = self._get_media_bytes(path)
             self.name = path.split('/')[-1]
-            if not self.name.endswith(('.jpg', '.jpeg', '.png', '.gif')):
-                self.name += '.jpg'
+            if not self.name.endswith(('.jpg', '.jpeg', '.png', '.gif', '.mp3', '.mp4')):
+                raise ValueError("Supported file types for custom names: jpg, jpeg, png, gif, mp3, mp4")
```
Comment (Makuh17) [inline src/rapidata/rapidata_client/assets/media_asset.py:49]: we do not support .webp? I am not really sure where we support it and where not tbh

**Review by Makuh17 — APPROVED** [review]
Couple of suggestions to consider, but otherwise looks good

File: docs/examples/compare_order.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
 # Example Compare Order
 
-[:octicons-arrow-right-24: Introduction to custom orders](../custom_orders.md)
+This example builds on what was introduced in the [Quickstart](quickstart.md) as well as the [Improve Order Quality](improve_order_quality.md) guide.
+
+## Order description
+
+We have to state of the art models, Flux and Midjourney that generate images based on a description aka prompt. We want to now find out, which of the images more closely aligns with the prompt.
```
Comment (kannwism) [inline docs/examples/compare_order.md:7]:
```suggestion
We have two state of the art models, Flux and Midjourney that generate images based on a description aka prompt. We want to find out which of the images more closely aligns with the prompt. 
```

File: docs/improve_order_quality.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+A validation set is a collection of tasks with known answers. It provides labelers with examples of how you want your data annotated, offering immediate feedback to align their work with your requirements.
+
+### How does it work?
+
+The validation task has a known truth and will be shown infront of the actual labeling task. Think of it as an interview question before the work starts. The labelers are only allowed to proceed to the actual task if they correctly solve one validation task.
+
+## Example Validation Set
+
+Let’s walk through how to create and use a validation set using the **Comparison Order** as an example. The principles apply to any order type.
```
Comment (kannwism) [inline docs/improve_order_quality.md:15]:
```suggestion
Let’s walk through how to create and use a validation set using the **Compare Order** as an example. The principles apply to any order type.
```

File: docs/improve_order_quality.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+1. Create an empty validation set builder with a name to later find it again:
+
+```py
+validation_set_builder = rapi.new_validation_set(
+        "Example Compare Validataion Set")
+```
+
+2. Now that we have the validataion set, we can start adding tasks to it, where we know the ground truth. In our case, each task is called a 'Rapid'. Let's first create the Rapid, and then add it to the validation set:
```
Comment (kannwism) [inline docs/improve_order_quality.md:37]:
```suggestion
2. Now that we have the validation set builder, we can start adding tasks to it, where we know the ground truth. In our case, each task is called a 'Rapid'. Let's first create the Rapid, and then add it to the validation set builder:
```

File: src/rapidata/rapidata_client/assets/media_asset.py (hunk 2 above)
Comment (kannwism) [reply src/rapidata/rapidata_client/assets/media_asset.py:49]: I think webp should work fine

**Review by kannwism — APPROVED** [review]

File: src/rapidata/rapidata_client/assets/media_asset.py (hunk 2 above)
Comment (LinoGiger) [reply src/rapidata/rapidata_client/assets/media_asset.py:49]: good catch! thanks!

**Pull request merged**